backend/Pipfile

```diff
@@ -20,7 +20,7 @@ dj-database-url = "*"
 djangorestframework = "*"
 psycopg2 = "*"
 sentry-sdk = "*"
-django = "==3.1.7"
+django = "==3.2.4"
```

Comment:
In general, please don't edit backend dependencies in a frontend change unless it's relevant to the frontend change. 

You should revert this change and the lock file change